static/js/server_events_dispatch.js

```diff
@@ -143,6 +143,7 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
                 emails_restricted_to_domains: noop,
                 video_chat_provider: compose.update_video_chat_button_display,
                 waiting_period_threshold: noop,
+                wildcard_mention_policy: noop,
```

Comment:
This is unrelated to your change, but I wonder why we need all these `noop` lines.  I guess explicit is better, but if the most common case is to to do no extra processing, I wonder why we don't just change the code to skip the post-processing step if the field's not present in this dict.